### DISCUSSION TITLE
Inconsistant features and vimp values

### DISCUSSION THREAD

**Discussion by jianxinwang** [kogalur/randomForestSRC#381] [Q&A]
Each time I run my code to fit a model and run subsample to get variable confidence interval, I get slightly different results. I tried to set the random seed, but this does not have any effect. Any thoughts on this? Thanks in advance!

**Comment by kogalur** [comment]
Please see:

https://github.com/kogalur/randomForestSRC/issues/67#issuecomment-1100196455

**Reply by jianxinwang** [reply]
Thanks for your reply. I just read that thread. Is the seed option in place yet? Or I have to wait?

**Comment by kogalur** [comment]
No, we have no plans for that.  See:

https://github.com/kogalur/randomForestSRC/issues/67#issuecomment-1424831686

Your work around is to run in serial mode.  It "might" work.  It's a complicated analysis.  Your best bet is to try it.
